cores/arduino/HardwareTimer.h

```diff
@@ -131,13 +134,13 @@ class HardwareTimer {
     // The following function(s) are available for more advanced timer options
     TIM_HandleTypeDef *getHandle();  // return the handle address for HAL related configuration
   private:
-    TIM_OC_InitTypeDef _channelOC[TIMER_CHANNELS];
-    TIM_IC_InitTypeDef _channelIC[TIMER_CHANNELS];
+    uint32_t _ChannelMode[TIMER_CHANNELS];
```

Comment:
Should be `TimerModes_t`?
```suggestion
    TimerModes_t _ChannelMode[TIMER_CHANNELS];
```

Reply:
Done